### DISCUSSION TITLE
Downloads

### DISCUSSION THREAD

**Discussion by Trululuexe** [z-huang/music#1054] [General]
Hello! Where are the downloads stored and which file are they, becausd i would juste take mp3 files to play them some where else like offline on my laptop, thx!

**Comment by z-huang** [comment]
Downloads are saved in chunks in the app-specific storage, so you can't access them and play them directly.

**Comment by Trululuexe** [comment]
Can you do that differently?

**Reply by z-huang** [reply]
Short answer: no.

**Comment by Trululuexe** [comment]
just why?

**Reply by z-huang** [reply]
The app is using ExoPlayer to play songs. For downloading content, see https://developer.android.com/guide/topics/media/exoplayer/downloading-media. That's why it's stored into chunks.
That said, you can make a pull request if you can find other ways to download media and export to mp3.

**Reply by Trululuexe** [reply]
ok
